docs/cli/graphql-transformer/directives.md

```diff
@@ -1367,8 +1374,8 @@ The @function directive allows you to quickly connect lambda resolvers to an App
 Let's assume you have deployed an *echo* function with the following contents:
 
 ```javascript
-exports.handler = function (event, context) {
-  context.done(null, event.arguments.msg);
+exports.handler = async event => {
```

Comment:
Can we use the idiomatic `async function(event, context){...}` style from the Lambda docs? https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html

Reply:
changed